Makefile

```diff
@@ -1,3 +1,5 @@
+dev_build_version=$(shell git describe --always --dirty)
```

Comment:
I can add `--tags`, and on a tag it will show the tag, and then on a non-tag commit it will display `dev build v1.0.0-12-gc162649-dirty`. It will list the previous tag, number of commits away from it, and the commit hash. It will also add on `dirty` if the working directory is changed.

Reply:
I added this in 2a6a598.